Changelog.md

```diff
@@ -9,6 +9,7 @@ Compiler Features:
 Bugfixes:
  * SMTChecker: Fix internal error on mapping access caused by too strong requirements on sort compatibility of the index and mapping domain.
  * SMTChecker: Fix internal error when using bitwise operators with an array element as argument.
+ * DockerFiles: Added parsec and tabulate to base dockerfiles
```

Comment:
```suggestion
```

A change in a Dockerfile doesn't warrant a changelog entry; only changes that are user visible.

Reply:
Updated